dask/delayed.py

```diff
@@ -230,7 +231,14 @@ def tokenize(*args, **kwargs):
 
 
 @curry
-def delayed(obj, name=None, pure=None, nout=None, traverse=True):
+def delayed(
+    obj: Any,
+    name: Hashable = None,
+    pure: Optional[bool] = None,
+    nout: Optional[int] = None,
+    traverse: bool = True
+) -> "Delayed":
```

Comment:
```suggestion
) -> Delayed:
```
+ add at the top of the module:
```
from __future__ import annotations
```